src/main/kotlin/org/jitsi/jibri/selenium/pageobjects/CallPage.kt

```diff
@@ -82,10 +82,24 @@ class CallPage(driver: RemoteWebDriver) : AbstractPageObject(driver) {
         val result = driver.executeScript("""
             try {
                 window._jibriParticipants = [];
+                const existingMembers = APP.conference._room.room.members || {};
+                console.log("There were " + Object.keys(existingMembers).length + " existing members");
+                for (let key in existingMembers) {
+                    const existingMember = existingMembers[key];
+                    if (existingMember.identity) {
+                        console.log("Member ", existingMember, " has identity, adding");
+                        window._jibriParticipants.push(existingMember.identity);
+                    } else {
+                        console.log("Member ", existingMember.jid, " has no identity, skipping");
+                    }
+                }
                 APP.conference._room.room.addListener(
                     "xmpp.muc_member_joined",
                     (from, nick, role, hidden, statsid, status, identity) => {
-                        window._jibriParticipants.push(identity);
+                        console.log("Jibri got MUC_MEMBER_JOINED: ", from, identity);
+                        if (identity) {
+                            window._jibriParticipants.push(identity);
+                        }
```

Comment:
So the identity truthy check wasn't done before. What does the truthy check give you?

Reply:
previously 'null' could end up in this list, which we don't want.